src/sentry/api/endpoints/organization_member/index.py

```diff
@@ -369,6 +369,31 @@ def post(self, request: Request, organization) -> Response:
             )
             return Response({"detail": ERR_RATE_LIMITED}, status=429)
 
+        is_member = not request.access.has_scope("member:admin") and (
+            request.access.has_scope("member:invite")
+        )
+        disable_invites_to_other_teams = (
+            not organization.flags.allow_joinleave and not organization.flags.disable_member_invite
+        )
+        has_team_roles = "teamRoles" in result and bool(result["teamRoles"])
```

Comment:
is it possible to assign a member to a team and not assign them a team role? i think team roles are only available to certain plans -- should this be checking result["teams"] instead?

Reply:
I wasn't sure because of this comment 😭 

https://github.com/getsentry/sentry/blob/82756cde16e0b8f8ea094fc245b98a460b389c6b/src/sentry/api/endpoints/organization_member/index.py#L87-L89

Reply:
oh interesting ok, ignore me! i guess the role could be None?